### PR TITLE
Doc: translated mobile apps configuration in french

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | yes
| Fixed tickets | N/A
| License       | MIT

I added 🇫🇷 translation for mobile apps configuration. 

@jlnostr @Strubbl: need your help for :de: translation.
@matteocoder: need your help for :it: translation.

### CODE REVIEW

**Comment by matteocoder** [comment]
Got it!

**Comment by j0k3r** [comment]
I'm wondering why the itatlian translation is under `ita` and not just `it`?

File: docs/de/user/configuring_mobile.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Konfiguration mobiler Apps für wallabag
+=======================================
+
+Schritte zum Konfigurieren der App
+----------------------------------
+
+- als erstes *neuen Client erstellen* in dem Abschnitt ``API Client Management`` wählen. Der Name deiner App ist nicht so wichtig wie die ``Client-ID`` und das ``Client-Secret``. Schreib diese zwei Werte auf.
```
Comment (jlnostr) [inline docs/de/user/configuring_mobile.rst]: @Strubbl The first sentence doesn't look like a sentence, the rest is looking good 👍

File: docs/de/user/configuring_mobile.rst [hunk 2]
```diff
@@ -4,10 +4,9 @@ Konfiguration mobiler Apps für wallabag
 Schritte zum Konfigurieren der App
 ----------------------------------
 
-- als erstes *neuen Client erstellen* in dem Abschnitt ``API Client Management`` wählen. Der Name deiner App ist nicht so wichtig wie die ``Client-ID`` und das ``Client-Secret``. Schreib diese zwei Werte auf.
-- Wenn du eine mobile App herunterlädst, wird sie dich nach der Server-Adresse deiner wallabag Instanz fragen. Zum Beispiel ist es ``https://app.wallabag.it`` bei wallabag.it.
-- Sie wird dich auch nach den zuvor genannten ``Client-ID`` und ``Client-Secret`` fragen. Bitte trage die in die Textfelder ein, wenn du danach gefragt wirst.
+- Als Erstes erstellst du einen neuen *neuen Client* in dem Abschnitt ``API Client Management``. Der Name deiner App ist dabei nicht so wichtig wie die ``Client-ID`` und das ``Client-Secret``. Schreib diese zwei Werte auf.
```
Comment (Strubbl) [inline docs/de/user/configuring_mobile.rst]: > neuen *neuen

double "neuen"

**Pull request merged**